src/config.c

```diff
@@ -609,6 +612,7 @@ void *parse_window_shader_prefix(const char *src, const char **end, void *user_d
 	char *tmp = (char *)trim_both(untrimed_shader_source, &length);
 	tmp[length] = '\0';
 	char *shader_source = NULL;
+	free(untrimed_shader_source);
```

Comment:
this is unnecessary. `untrimed_shader_source` is automatically freed when it's out of scope.